osfoffline/filesystem_manager/osf_event_handler.py

```diff
@@ -61,8 +61,8 @@ def on_moved(self, event):
                     # This means it was put into a place on the hierarchy being watched but otherwise not attached to a
                     # node, so it needs to be added just like a new event rather than as a move.
 
-                    new_event = DirCreatedEvent(event.dest_path) if event.is_directory else FileCreatedEvent(event.dest_path)
-                    yield from self._create_file_or_folder(new_event, src_path=dest_path)
+                    # new_event = DirCreatedEvent(event.dest_path) if event.is_directory else FileCreatedEvent(event.dest_path)
+                    # yield from self._create_file_or_folder(new_event, src_path=dest_path)
```

Comment:
This change does not appear to reference either 5129 or 5131. What ticket is covered by these changes?